content/en/docs/languages/java/automatic/configuration.md

```diff
@@ -126,6 +117,259 @@ The Java agent logging mode. The following 3 modes are supported:
+| `OTEL_EXPORTER_ZIPKIN_ENDPOINT` | The Zipkin endpoint to connect to. Default is `http://localhost:9411/api/v2/spans`. Currently only HTTP is supported. |
+
+### Prometheus exporter
+
+The
+[Prometheus](https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md)
+exporter is only available for the metric signal.
+
+| Environment variable               | Description                                                                        |
+| ---------------------------------- | ---------------------------------------------------------------------------------- |
+| `OTEL_METRICS_EXPORTER=prometheus` | Select the Prometheus exporter                                                     |
+| `OTEL_EXPORTER_PROMETHEUS_PORT`    | The local port used to bind the prometheus metric server. Default is `9464`.       |
+| `OTEL_EXPORTER_PROMETHEUS_HOST`    | The local address used to bind the prometheus metric server. Default is `0.0.0.0`. |
+
+Note that this is a pull exporter - it opens up a server on the local process
+listening on the specified host and port, which a Prometheus server scrapes
+from.
+
+### Logging exporter
+
+The logging exporter prints the name of the span along with its attributes to
+stdout. It's mainly used for testing and debugging.
+
+| Environment variable            | Description                             |
+| ------------------------------- | --------------------------------------- |
+| `OTEL_TRACES_EXPORTER=console`  | Select the logging exporter for tracing |
+| `OTEL_METRICS_EXPORTER=console` | Select the logging exporter for metrics |
+| `OTEL_LOGS_EXPORTER=console`    | Select the logging exporter for logs    |
+
+The logging exporter is also set when `OTEL_TRACES_EXPORTER`,
+`OTEL_METRICS_EXPORTER`, or `OTEL_LOGS_EXPORTER` is set to `logging`. `logging`
+is a deprecated alias for `console`, the preferred value as
+[defined in the specification](/docs/specs/otel/configuration/sdk-environment-variables/#exporter-selection).
+
+### Logging OTLP JSON exporter
+
+The logging-otlp exporter writes the telemetry data to the JUL logger in OTLP
+JSON form. It's a more verbose output mainly used for testing and debugging.
+
+| Environment variable                 | Description                                       |
+| ------------------------------------ | ------------------------------------------------- |
+| `OTEL_TRACES_EXPORTER=logging-otlp`  | Select the logging OTLP JSON exporter for tracing |
+| `OTEL_METRICS_EXPORTER=logging-otlp` | Select the logging OTLP JSON exporter for metrics |
+| `OTEL_LOGS_EXPORTER=logging-otlp`    | Select the logging OTLP JSON exporter for logs    |
+
+**NOTE:** While the `OtlpJsonLogging{Signal}Exporters` are stable, specifying
+their use via `OTEL_{signal}_EXPORTER=logging-otlp` is experimental and subject
+to change or removal.
```

Comment:
Switch to the admonitions format we use in the docs. For example: https://github.com/open-telemetry/opentelemetry.io/blob/05a7dfa420911704c8a234a1783555e431b5ef50/content/en/ecosystem/integrations.md?plain=1#L29-L41